ivy/functional/frontends/numpy/fft/discrete_fourier_transform.py

```diff
@@ -125,3 +124,16 @@ def rfftfreq(n, d=1.0):
     N = n // 2 + 1
     results = ivy.arange(0, N, dtype=int)
     return results * val
+
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"1.24.3 and above": ("float16",)}, "numpy")
```

Comment:
Have you tried running this locally? I don't think 'float16` should be the only supported dtype.